src/main/java/org/moddingx/modlistcreator/changelogger/Changelogger.java

```diff
@@ -0,0 +1,54 @@
+package org.moddingx.modlistcreator.changelogger;
+
+import joptsimple.OptionException;
+import joptsimple.OptionParser;
+import joptsimple.OptionSet;
+import joptsimple.OptionSpec;
+import joptsimple.util.PathConverter;
+import joptsimple.util.PathProperties;
+import org.moddingx.modlistcreator.output.OutputTarget;
+import org.moddingx.modlistcreator.util.CommonUtils;
+import org.moddingx.modlistcreator.util.EnumConverters;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+
+public class Changelogger {
+
+    public static void run(String[] args) throws IOException {
+        OptionParser options = new OptionParser();
+        OptionSpec<Path> specOld = options.acceptsAll(List.of("o", "old"), "Defines the old modpack zip or json file").withOptionalArg().withValuesConvertedBy(new PathConverter(PathProperties.FILE_EXISTING)).defaultsTo(Paths.get("old.json"));
+        OptionSpec<Path> specNew = options.acceptsAll(List.of("n", "new"), "Defines the new modpack zip or json file").withOptionalArg().withValuesConvertedBy(new PathConverter(PathProperties.FILE_EXISTING)).defaultsTo(Paths.get("new.json"));
+        OptionSpec<String> specOutput = options.accepts("output", "Defines the output file name without extension").withOptionalArg().ofType(String.class).defaultsTo("changelog");
```

Comment:
For full file names, extension should be part of the file name as well. (Is a default value even needed here?)

Reply:
You can't define the correct file extension if you select multiple output types. And I'd say yes, a default name is quiet useful.